### ISSUE TITLE
9.2.4.1 Bereiche überspringbar: Cookie-Einstellungen sollen vor Skiplinks im Kopfbereich erreicht werden

### ISSUE THREAD

**Issue by sprungmarker** [BIK-BITV/BIK-Web-Test#454]
Von der BFIT kam die Anforderung für uns rein, dass:

- wenn es Cookie-Einstellungen auf der Seite gibt und
- wenn es auch einen oder mehrere Skiplinks im Kopfbereich der Seiten gibt
- dann sollten die Cookie-Einstellungen ### VOR den Skiplinks per Tastatur erreicht werden

Das widerspricht:
> Sprunglinks am Seitenbeginn sind die ersten fokussierbaren Elemente der Tabreihenfolge

**Comment by Michael-Detmers** [comment]
Die Argumentation war, dass der Cookie-Hinweis wichtige Einstellungen für den Auftritt enthält, was die Nutzererfahrung und die Einhaltung der Datenschutzbestimmungen beeinträchtigen kann.

Für solche Fälle hätte ich am liebsten einen zugänglichen modalen Dialog, der oft aber explizit nicht eingebaut werden soll.

Sind die Cookie-Einstellungen für die Bedienung der Seite unnötig, sehe ich keinen Grund mit der Ausgabe vor den Skip-Links damit zu stören.

**Comment by detlevhfischer** [comment]
> Sprunglinks am Seitenbeginn sind die ersten fokussierbaren Elemente der Tabreihenfolge

Ich denke, dieser Hinweis wurde vor dem verbreiteten Auftauchen der Cookie-Banner geschrieben (hoffentlich finden sich da bald mal weniger nervige Lösungen). Ich gucke mal, wie wir den Text anpassen können.

**Comment by sweckenmann** [comment]
@Michael-Detmers 

> Sind die Cookie-Einstellungen für die Bedienung der Seite unnötig, sehe ich keinen Grund mit der Ausgabe vor den Skip-Links damit zu stören.

Wann sind den Cookie-Einstlelungen für die Bedienung der Seite unnötig?

**Comment by sweckenmann** [comment]
Danke @detlevhfischer für den PR. 

> Sprunglinks am Seitenbeginn sind die ersten fokussierbaren Elemente der
  Tabreihenfolge (oder ggf. die ersten nach einem Cookie-Bannner oder Dialog)
  
Das "oder" lässt dann beide Möglichkeiten zu. D.h. Skiplinks vor dem Cookie-Dialog wäre noch kein FAIL (wo ich mitgehen kann). 

@sprungmarker Fordert das BFIT das für Konformität zwingend?

**Comment by sprungmarker** [comment]
>> @sprungmarker Fordert das BFIT das für Konformität zwingend?
@sweckenmann Für uns gilt so eine Anforderung der BFIT als zwingend intern für unser Testing.

**Comment by sweckenmann** [comment]
@sprungmarker Ja, das verstehe ich. Würde mich nur interessieren, ob die Überwachungsstelle des Bundes dafür ein FAIL geben würde oder ob sie es euch als Best Practice (die ihr umsetzen sollt und somit auch tut) mitgegeben hat.

**Comment by johannesFischer84** [comment]
Der Hinweis beim Prüfschritt 9.2.4.1 ist sicher in Ordnung, auch mit dem "oder". Als Fehler würde ich es dann aber nicht in 9.2.4.1 bewerten, sondern in 9.2.4.3 (Fokusreihenfolge) oder sogar in 9.1.3.2 (Lesereihenfolge).
Denn in der Lese-/Fokusreihenfolge liegt die eigentliche Ursache, denke ich. Der Cookie-Dialog sollte gelesen/erfasst werden können, bevor man zu den Sprunglinks gelangt. Die Funktionsweise der Sprunglinks kann trotzdem gegeben sein.

**Comment by sprungmarker** [comment]
@sweckenmann @Michael-Detmers ist es Pflicht, für die BFIT das abzuarbeiten oder nur eine Empfehlung?
